components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtDBQueries.java

```diff
@@ -22,7 +22,9 @@
  * This class contains default SQL queries
  * <p/>
  * TODO : Make the queries configurable from a file TODO : Use transactions and joins
+ * SQL queries are internal implementation of the DAO layer and should not be exposed to components outside
  */
+@Deprecated
```

Comment:
Add a comment stating what to use hence this is deprecated.

Reply:
Adressed in https://github.com/wso2/carbon-identity-framework/pull/4483